distrho/DistrhoUI_macOS.mm

```diff
@@ -14,6 +14,8 @@
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
+#if !DISTRHO_PLUGIN_HAS_EXTERNAL_UI
```

Comment:
this macro is at the wrong place, there is no includes ever here, so this for sure will break normal build.

I am not sure if this is the correct approach anyway, we need to see if we can skip this file altogether in the makefiles.

Reply:
ah yes you are right, I overlooked this (I had the define in the Makefile). So we need a different approach.... The problem is, that the defines are in the header. So perhaps we just could include the header with the defines here before the `#if`?

Reply:
There is a UI_TYPE=external in the makefile for the plugin that is needed, we can just use that as comparison

Reply:
ah yes, I forgot this... I will then move the `if` to the Makefile....